test/BaselineOutput/SingleDebug/MulticlassLogisticRegression/MulticlassLogisticRegression-TrainTest-iris-tree-featurized-out.txt

```diff
@@ -51,19 +51,19 @@ Virtual memory usage(MB): %Number%
 [5] 'LBFGS data prep' started.
 [5] 'LBFGS data prep' finished in %Time%.
 [6] 'LBFGS Optimizer' started.
-[6] (%Time%)	0 iterations	Loss: 1.0986123085022
-[6] (%Time%)	1 iterations	Loss: 0.62896740436554	Improvement: 0.4696
-[6] (%Time%)	2 iterations	Loss: 0.213765218853951	Improvement: 0.4261
-[6] (%Time%)	3 iterations	Loss: 0.144495338201523	Improvement: 0.1542
-[6] (%Time%)	4 iterations	Loss: 0.106832779943943	Improvement: 0.06646
-[6] (%Time%)	5 iterations	Loss: 0.0936193987727165	Improvement: 0.02649
-[6] (%Time%)	6 iterations	Loss: 0.083323560655117	Improvement: 0.01434
-[6] (%Time%)	7 iterations	Loss: 0.0787383615970612	Improvement: 0.007024
-[6] (%Time%)	8 iterations	Loss: 0.0768212750554085	Improvement: 0.003194
+[6] (%Time%)	0 iterations	Loss: 1.0986123085021973
```

Comment:
💭 This pull request is only still marked as a draft because I'm not sure how I feel about changing this baseline. I'm going to try and change only the SymSGD baseline and see if the tests still pass. The one here is known to be flaky even with my changes.

Reply:
This change was subsequently removed from the pull request.